### PR TITLE
container_names and links directive

### LINKED ISSUE #71: Add shorter container_name to each service in compose
Now the names are automatically generated, would be nice to add some shorter like `bitcoin`, `liquid`, `electrs_liquid`, `chopsticks` etc.. [using the docker-compose directive `container_name` ](https://docs.docker.com/compose/compose-file/#container_name)

### LINKED ISSUE #72: Drop link directive in compose
`depends_on` directive is enough to make each container reachable in the same local network, and also `link` is deprecated.

**What to do?**
Remove the `link` directive in each `service` in the `docker-compose-regtest.yml` and `docker-compose-regtest-liquid.yml `

### LINKED ISSUE #74: Add Nigiri Travis reference in documentation
We have a completely different version of nigiri to be used inside Travis and other CI services. So the readme should ask "Are you looking to spin-up Nigiri in Travis or Github Action? Look here" and link to https://github.com/vulpemventures/nigiri-travis

### PR DESCRIPTION
**Pull request by francismars**
- Added container_names: it closes #71 
- Removed deprecated links directive: it closes #72
- Added Nigiri Travis reference in documentation: it closes #74 
- Updated container_name related documentation
- Added necessary dependencies

### CODE REVIEW

**Comment by francismars** [comment]
updated the documentation to reflect the changes on container_names

File: resources/docker-compose-regtest.yml [hunk 1]
```diff
@@ -77,9 +77,6 @@ services:
     networks:
       local:
         ipv4_address: 10.10.0.13
-    links:
-      - bitcoin
-      - electrs
     depends_on:
```
Comment (tiero) [inline resources/docker-compose-regtest.yml:80]: Here it should depend on bitcoin too. Add - bitcoin here under the depends_on directive

File: resources/docker-compose-regtest-liquid.yml [hunk 2]
```diff
@@ -159,9 +156,6 @@ services:
       - 0.0.0.0:3000
       - --chain
       - liquid
-    links:
-      - electrs-liquid
-      - liquid
     depends_on:
       - electrs-liquid
```
Comment (tiero) [inline resources/docker-compose-regtest-liquid.yml:160]: same here. Add - liquid under depends_on

File: resources/docker-compose-regtest-liquid.yml [hunk 3]
```diff
@@ -134,9 +133,6 @@ services:
       - 10.10.0.12:3002
       - --addr
       - 0.0.0.0:3000
-    links:
-      - electrs
-      - bitcoin
     depends_on:
       - electrs
```
Comment (tiero) [inline resources/docker-compose-regtest-liquid.yml:137]: same here, add - bitcoin  under depends_on

**Review by tiero — CHANGES_REQUESTED** [review]
Address the comments

**Comment by tiero** [comment]
Please @francismars add the "it closes <Github Issue URL>" for each ticket you are asking to merge. Thanks

**Review by tiero — APPROVED** [review]

**Pull request merged**